LaunchDarklyAndroidBinding/Transforms/Metadata.xml

```diff
@@ -1,10 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-    <!--
-  This sample removes the class: android.support.v4.content.AsyncTaskLoader.LoadTask:
-  <remove-node path="/api/package[@name='android.support.v4.content']/class[@name='AsyncTaskLoader.LoadTask']" />
-  
-  This sample removes the method: android.support.v4.content.CursorLoader.loadInBackground:
-  <remove-node path="/api/package[@name='android.support.v4.content']/class[@name='CursorLoader']/method[@name='loadInBackground']" />
-  -->
+
+<!--    <remove-node path="/api/package[@name='com.launchdarkly.android.gson']/class[@name='LDFailureSerialization']"/>-->
```

Comment:
Are we trying to remove these or not? If not can we explain with a comment why this is here?

Reply:
Oh that's leftover commented code that should have been removed. I just removed it.